libs/api/domains/finance/src/lib/api-domains-finance.resolver.ts

```diff
@@ -217,4 +218,18 @@ export class FinanceResolver {
       user,
     )
   }
+
+  @Query(() => DebtLessCertificateModel)
+  @Audit()
+  @Scopes(ApiScope.financeOverview, ApiScope.internal)
```

Comment:
Why the need of the internal scope over using the already applied financeOverview scope?

Reply:
You didn't answer the question.

Reply:
Ahh I see. No we should keep it this way for now. It is another discussion if we should be re-using the client id between the service-portal and the application-system.